ui/dashboard.js

```diff
@@ -78,10 +78,32 @@ function create_box (id, icon, number, currency_icon) {
     settings.page_index = $('#page-wrapper').html();
 }
 
+function update_eth_node_conn_status_ui(is_local_eth) {
+    let str;
+    if (!is_local_eth) {
+        str='<a class="eth-node-status-link" href="#"><i class="fa fa-unlink fa-fw"></i></a><ul class="dropdown-menu dropdown-eth-status alert-warning"><p>Not connected to a local ethereum node</p></ul>';
+    } else {
+        str='<a class="eth-node-status-link" href="#"><i class="fa fa-link fa-fw"></i></a><ul class="dropdown-menu dropdown-eth-status alert-success"><p>Connected to a local ethereum node</p></ul>';
+    }
+    $('.eth-node-status').html(str);
+}
 
 
 var alert_id = 0;
 function add_alert_dropdown(alert_text, alert_time) {
+    if (
```

Comment:
also for the future: I guess there is no other way right now to see if we are connected to an ethereum node, but reading the alert text is hacky. Should properly expose it from the backend.

Reply:
Yes, I understand and agree. How would I go about doing that?